tests/examples/custom_components.py

```diff
@@ -0,0 +1,152 @@
+import dash_ag_grid as dag
+from dash import Dash, html, dcc, Input, Output, State, ctx
+import dash_bootstrap_components as dbc
```

Comment:
looks like we need to add `dbc` to `tests/requirements.txt`
https://app.circleci.com/pipelines/github/plotly/dash-ag-grid/18/workflows/33d9a3e9-8c1a-4747-a170-7fa3f443d7e6/jobs/18

Reply:
Yeah, I am removing it now.

Reply:
this dependency was removed in https://github.com/plotly/dash-ag-grid/pull/39/commits/0ea7b8c1c4f010ee3c4c1ff052baa295f5b34938